### PR TITLE
OPHJOD-540: Update color tokens

### PR DESCRIPTION
**Pull request by juhaniko**
## Description

Updated Tailwind CSS color tokens.

## Related JIRA ticket

https://jira.eduuni.fi/browse/OPHJOD-540

### CODE REVIEW

File: tailwind.config.ts [hunk 1]
```diff
@@ -10,22 +10,25 @@ export default {
     },
     // https://www.figma.com/file/6M2LrpSCcB0thlFDaQAI2J/cx_jod_client?node-id=542%3A6241
```
Comment (sauanto) [inline tailwind.config.ts:11]: The link should be updated to the new version.

**Review by sauanto — CHANGES_REQUESTED** [review]
lib/docs/Colors.mdx should be also updated to the new version.

File: tailwind.config.ts [hunk 2]
```diff
@@ -10,22 +10,25 @@ export default {
...
+      accent: '#006db3',
+      'secondary-1': '#47a7e2',
+      'secondary-2': '#1ab1bb',
+      'secondary-3': '#f08958',
+      'secondary-4': '#d260bb',
+      'secondary-5': '#dcdcdc',
       white: '#ffffff',
       black: '#000000',
       'primary-gray': '#333333',
       'secondary-gray': '#767676',
```
Comment (ketsappi) [inline tailwind.config.ts:22]: It seems that this is also changed in the design tokens:
`#6E6E6E`

File: tailwind.config.ts (hunk 1 above)
Comment (juhaniko) [reply tailwind.config.ts]: Link updated

File: tailwind.config.ts (hunk 2 above)
Comment (juhaniko) [reply tailwind.config.ts]: It's updated now as well

**Review by sauanto — APPROVED** [review]

**Pull request merged**